server/boosting.lua

```diff
@@ -132,12 +132,19 @@ local function SpawnCar(src)
         SetVehicleDoorsLocked(car, 2)
 
         local plate = GeneratePlate()
-        ActivePlates[plate] = math.random(3, 10)
+        ActivePlates[plate] = true
+
+        Entity(car).state.Boosting = {
+            boostHacks = math.random(3, 10),
+            boostCooldown = false
+        }
+
+        exports['Renewed-Fuel']:SetFuel(car, 100.0)

```

Comment:
Before merge I think you should change the export on with fuel state bag (ox fuel) or make it client side for people with legacy fuel